mega-linter-runner/lib/runner.js

```diff
@@ -113,13 +113,12 @@ ERROR: Docker engine has not been found on your system.
 
     // Build docker run options
     const lintPath = path.resolve(options.path || ".");
-    const commandArgs = [
-      "run",
-      "-v",
-      "/var/run/docker.sock:/var/run/docker.sock:rw",
-      "-v",
-      `${lintPath}:/tmp/lint:rw`,
-    ];
+    const commandArgs = [ "run" ];
+    if (options.containername) {
+      commandArgs.push(...["--name", options.containername]);
+    }
+    commandArgs.push(...["-v", "/var/run/docker.sock:/var/run/docker.sock:rw"]);
+    commandArgs.push(...["-v", `${lintPath}:/tmp/lint:rw`]);  
```

Comment:
Nitpick: I like the clarity of keeping each argument on its own line; this clarity can be retained with marginally less code in a single push:

```js
commandArgs.push(…[
  "-v", "/var/run/docker.sock:/var/run/docker.sock:rw",
  "-v", `${lintPath}:/tmp/lint:rw`
]);
```